core/src/components/tab-button/tab-button.tsx

```diff
@@ -107,6 +116,22 @@ export class TabButton implements ComponentInterface, AnchorInterface {
     }
   }
 
+  private getShape(): string | undefined {
+    const theme = getIonTheme(this);
+    const { shape } = this;
+
+    // TODO(ROU-11300): Remove theme check when shapes are defined for all themes.
```

Comment:
When copying over code, let's verify what exactly is being copied. This ticket is not associated with tab-button. Make a Jira ticket for tab-button.